models/repo.go

```diff
@@ -1468,7 +1475,15 @@ func GetUserRepositories(userID int64, private bool, page, pageSize int) ([]*Rep
 // GetUserRepositories returns a list of mirror repositories of given user.
 func GetUserMirrorRepositories(userID int64) ([]*Repository, error) {
 	repos := make([]*Repository, 0, 10)
-	return repos, x.Where("owner_id = ?", userID).And("is_mirror = ?", true).Find(&repos)
+	sess := x.Where("owner_id = ?", userID).And("is_mirror = ?", true).Desc("uid")
+
+	if setting.UsePostgreSQL {
```

Comment:
Just look at the XORM docs, or maybe @lunny can give you a hint since he's the XORM author ;)